pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -803,6 +805,14 @@ def parse_args_and_make_package(args=None):
     ap.add_argument('--manifest-placeholders', dest='manifest_placeholders',
                     default='[:]', help=('Inject build variables into the manifest '
                                          'via the manifestPlaceholders property'))
+    ap.add_argument('--network-security-config', dest='network_security_config', default='',
+                    help='Add a Network Security Configuration file path to AndroidManifest.xml')
+    ap.add_argument('--uses-cleartext-traffic', dest='uses_cleartext_traffic', default='',
+                    help='Indicate that app intends to use cleartext network traffic in AndroidManifest.xml')
+    ap.add_argument('--service-class-name', dest='service_class_name', default='org.kivy.android.PythonService',
```

Comment:
I feel like the `'org.kivy.android.PythonService'` string should probably be pulled out to a named constant at the top of the file, and the same for the activity.

Reply:
good point!

Reply:
done